src/logging.cr

```diff
@@ -1,15 +1,37 @@
-# Application dependencies
-require "log_helper"
 require "placeos-log-backend"
+require "raven"
+require "raven/integrations/action-controller"
 
-# Application code
 require "./constants"
 
 # Logging configuration
-log_level = App.production? ? Log::Severity::Info : Log::Severity::Debug
-log_backend = PlaceOS::LogBackend.log_backend
+module PlaceOS::Triggers
```

Comment:
Could this go into a more isolate namespace or any of this inlined so that these aren't exposed anywhere else this top level module is reopened?